src/ai/ai_gui.cpp

```diff
@@ -270,4 +270,3 @@ void ShowAIConfigWindow()
 	CloseWindowByClass(WC_GAME_OPTIONS);
 	new AIConfigWindow();
 }
-
```

Comment:
I don't think you mean to get rid of the newline at the end of the file.

Reply:
Strange, Visual Studio shows that line is present.
![image](https://user-images.githubusercontent.com/43006711/222446089-ea85a865-e4c5-4305-9476-d18b399c4215.png)

Reply:
AH! I see what's happening, it had 2 lines at the end, and I removed one. Github however omits the last line.

Reply:
Anyway if it was missing new line at end of file, commit checker would fail

Reply:
Ah, didn't realize GitHub hid the last line.